controllers/mirrorpeer_controller.go

```diff
@@ -58,6 +59,9 @@ const hubRecoveryLabel = "cluster.open-cluster-management.io/backup"
 //+kubebuilder:rbac:groups=addon.open-cluster-management.io,resources=managedclusteraddons/finalizers,verbs=*
 //+kubebuilder:rbac:groups=addon.open-cluster-management.io,resources=managedclusteraddons;clustermanagementaddons,verbs=*
 //+kubebuilder:rbac:groups=addon.open-cluster-management.io,resources=managedclusteraddons/status,verbs=*
+//+kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;update;patch;delete
```

Comment:
do we need all of these? we only get,create and update from what I see in the code.